nextcord/channel.py

```diff
@@ -1058,6 +1068,7 @@ async def edit(
         flags: ChannelFlags = ...,
         reason: Optional[str] = ...,
         default_sort_order: Optional[SortOrderType] = ...,
+        default_forum_layout: Optional[ForumLayoutType] = ...,
```

Comment:
Unlike `default_sort_order`, this is not nullable. It is strictly an integer (in this case, a forum layout type).